src/components/CardPlay/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useRef, useEffect } from "react";
+import { PlayCircleTwoTone, PauseCircleTwoTone } from '@ant-design/icons';
+import "./styles.css";
+
+    const CardPlay = (props) => {
+    const audioPlayer = useRef(null);
+    const [progress, setProgress] = useState(80);
+    const [title, setTitle] = useState("Bad Bunny - Si la veo a tu mamá");
+    const [countLikes, setCountLikes] = useState(0);
+    const [countDislikes, setCountDislikes] = useState(0);
+      
+    useEffect(() => {
+      console.log(progress, audioPlayer.current.duration);
+    }, [progress]);
+    
+    const [isPlaying, setIsPlaying] = useState(false);
+    const handleLike = (event) => {
+      setCountLikes(countLikes +1)
+    }
+    const handleDislike = (event) => {
+      setCountDislikes(countDislikes +1)
+    }
+  
+    const toggleAudio = () => {
+      if (isPlaying) {
+        audioPlayer.current.pause();
+        setIsPlaying(false);
+  } else {
```

Comment:
Cuidado con la indentación.
```suggestion
        } else {
```